app/models/exceeding_request.rb

```diff
@@ -0,0 +1,43 @@
+class ExceedingRequest < ApplicationRecord
+  belongs_to :account_user
+
+  validate :user_can_send_one_exceeding_request_per_period, on: :create
+  validates :amount, numericality: { greater_than_or_equal_to: 0 }
+  validates :status, inclusion: { in: [true, false] }, on: :update
+
+  def user
+    account_user.user
+  end
+
+  def account
+    account_user.account
+  end
+
+  def self.exceeding_requests_for(user)
+    joins(:account_user)
+      .joins('INNER JOIN roles ON roles.id = account_users.role_id')
+      .where('account_users.user_id' => user.id,
+             'roles.name' => 'owner',
+             'status' => nil)
+  end
+
+  private
+
+  def user_can_send_one_exceeding_request_per_period
+    current_day = Date.today
+    ExceedingRequest.where(created_at: current_day..current_day.next).each do |request|
```

Comment:
It would be nice to see `ExceedingRequest.where(created_at: current_day..current_day.next)` as a scope